Sources/SwiftGenKit/Stencil/JSONContext.swift

```diff
@@ -0,0 +1,41 @@
+//
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+
+//
+// See the documentation file for a full description of this context's structure:
+// Documentation/SwiftGenKit Contexts/Json.md
+//
+
+extension JSON.Parser {
+  public func stencilContext() -> [String: Any] {
+    let files = self.files
+      .sorted { lhs, rhs in lhs.name < rhs.name }
+      .map(map(file:))
+
+    return [
+      "files": files
+    ]
+  }
+
+  private func map(file: JSON.File) -> [String: Any] {
+    // `documents` is an array for compability reasons with YAML and we'll be updated to be a single document in a
+    // future release
```

Comment:
s/we'll/will/

Though that comment will likely need to change if you implement @djbe 's suggestion of having both `"document"` (new key introduced even before 7.0 so people can start migrate) and `"documents"` (compatibility)

```suggestion
    // `documents` is an array for compability reasons with YAML
    // and will be updated to be a single document in a future release
```